simulator/risc_v/riscv_instr.h

```diff
@@ -61,4 +61,10 @@ static inline std::ostream& operator<<( std::ostream& out, const RISCVInstr<T>&
     return out << rhs.string_dump();
 }
 
+enum base_t
```

Comment:
Although this enum should be removed, please pay attention to our naming conventions for types and variables.